scripts/target/mame/mess.lua

```diff
@@ -904,6 +904,7 @@ BUSES["HP_IPC_IO"] = true
 BUSES["KC"] = true
 BUSES["LPCI"] = true
 BUSES["M5"] = true
+BUSES["MC10"] = true
```

Comment:
This isn’t in alphabetical order.

Reply:
Should I resort the whole list? Or just find a better place for my one addition?

Reply:
I decided to sort the whole group.